ng/scriptGenerators/ansible/AnsibleScriptGenerator.ts

```diff
@@ -0,0 +1,158 @@
+﻿module armExplorer {
+
+    export class AnsibleScriptGenerator {
+        private script: string = "";
+        private actionsIndex: number = 0;
+        private resourceDefinition: any = {};
+        constructor(private resolver: ScriptParametersResolver, resourceDefinition: any) {
+            this.resourceDefinition = resourceDefinition;
+        }
+
+        getScript(cmdActionPair: RMCommandInfo): string {
+            let cmdParameters = this.resolver.getParameters();
+            let currentScript: string = "";
+            
+            currentScript += "- hosts: localhost\n";
+            currentScript += "  tasks:\n";
+
+            switch (cmdActionPair.cmd) {
+                case CmdType.Get: {
+                    currentScript += '    - name: GET ' + this.resolver.getActionName() + '\n';
+                    currentScript += '      azure_rm_resource_facts:\n';
+                    currentScript += this.yamlFromResourceId("        ");
+                    break;
+                }
+                case CmdType.New: {
+                    if (cmdActionPair.isSetAction) {
+                        currentScript += '    - name: SET ' + this.resolver.getActionName() + '\n';
+                        currentScript += '      azure_rm_resource:\n';
+                        currentScript += this.yamlFromResourceId("        ");
+                    }
+                    else {
+                        let newName: string = "New" + this.resolver.getResourceName();
+                        currentScript += '    - name: CREATE ' + this.resolver.getActionName() + '\n';
+                        currentScript += '      azure_rm_resource:\n';
+                        // XXX - resolve this
```

Comment:
is this is a todo? can you open issues to track this instead of commenting here? also at Line 84

Reply:
it's now resolved properly, so no need for todo.